src/styles/utils/_varibles.scss

```
@@ -0,0 +1,9 @@
+$calendarWidth: 707px;
+$weekDay: 
+  "mon" 0,
+  "tue" 101px,
```

calculate this value with size block & gap
day-margin: calculations....
```suggestion
  "tue" $day-margin,
```